shell/src/main/java/alluxio/cli/GetConf.java

```diff
@@ -22,15 +22,22 @@
 import org.apache.commons.cli.Options;
 import org.apache.commons.cli.ParseException;
 
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Map;
 import java.util.Map.Entry;
 
 /**
  * Utility for printing Alluxio configuration.
  */
 public final class GetConf {
-  private static final String USAGE = "USAGE: GetConf [--unit <arg>] [KEY]\n\n"
+  private static final String USAGE = "USAGE: GetConf [--unit <arg>] [--display-name] [-e] [KEY]"
+      + "\n\n"
       + "GetConf prints the configured value for the given key. If the key is invalid, the "
       + "exit code will be nonzero. If the key is valid but isn't set, an empty string is printed. "
+      + "If \"--display-name\" option is specified, both the key and value are printed. "
+      + "If \"-e\" option is specified, the key is interpret as an environment variable name and "
```

Comment:
what does -e w/o --display-name print?. only value or keyvaluepair. i suspect the former and in that case the comment needs to be updated

Reply:
Yes it is the former case. I will update the comment.